examples/workload_identity/README.md

```diff
@@ -0,0 +1,44 @@
+# Simple Zonal Cluster
+
+This example illustrates how to create a simple cluster.
```

Comment:
Please give a better explanation for examples. They should explain what the example is actually doing, don't just copy & paste an existing one.